avm/res/api-management/service/portalsetting/main.bicep

```diff
@@ -0,0 +1,36 @@
+metadata name = 'API Management Service Portal Settings'
+metadata description = 'This module deploys an API Management Service Portal Setting.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent API Management service. Required if the template is used in a standalone deployment.')
+param apiManagementServiceName string
+
+@description('Required. Portal setting name.')
+@allowed([
+  'delegation'
+  'signin'
+  'signup'
+])
+param name string
+
+@description('Optional. Portal setting properties.')
+param properties object = {}
+
+resource service 'Microsoft.ApiManagement/service@2021-08-01' existing = {
+  name: apiManagementServiceName
+}
+
+resource portalSetting 'Microsoft.ApiManagement/service/portalsettings@2021-08-01' = if (!empty(properties)) {
```

Comment:
I guess this condition should be part of the parent deployment?

Reply:
Moved it there and made the properties param of the child module required.